### ISSUE TITLE
Stack in Typescript

### ISSUE THREAD

**Issue by DanielRomeo** [FOSS-UCSC/FOSSALGO#643]
#### This issue is about [Algo/DS Name](link to resource for the Algo/DS)

-   [x] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [x] Assign this issue to me (I want to work on this)

   -   [x] I ensure that I am not already assigned to 2 or more issues
  
-   [x] I will send my PR only after I'm assigned to this issue by any maintainer
@Danushka96 @RavinduSachintha

**Comment by DanielRomeo** [comment]
May I be assigned to the issue that i have just created.

**Comment by RavinduSachintha** [comment]
due to you have already assigned to another issue, we can not assign you to this one now. when you finish that one, let's work on this.

**Comment by DanielRomeo** [comment]
Hi @RavinduSachintha or @Danushka96 please assign me to this.
